instructor/dsl/partial.py

```diff
@@ -170,7 +170,7 @@ def extract_json(
                     elif mode in {Mode.JSON, Mode.MD_JSON, Mode.JSON_SCHEMA}:
                         if json_chunk := chunk.choices[0].delta.content:
                             yield json_chunk
-                    elif mode == Mode.TOOLS:
+                    elif mode == Mode.TOOLS or mode == Mode.STRUCTURED_OUTPUTS:
```

Comment:
use a set like `elif mode in {Mode.JSON, Mode.MD_JSON, Mode.JSON_SCHEMA}`

Reply:
Sure, that makes it more consistent. Is there a reason why we should prefer a set over say a normal if-else? Is that just so that it's easier to integrate/make changes down the line